app/views/to_change_everything/_share_links.html.erb

```diff
@@ -0,0 +1,8 @@
+<% if locale == 'espanol' %>
+  <li><a href="<%= t('.bulk_get.url').html_safe %>"><span class="icon"></span> <%= t('.bulk_get.text').html_safe %></a></li>
```

Comment:
the `.` shortcut in the translations (e.g. `t('.bulk_get.url')`) appends the view path.

since this files path has changed from `app/views/layouts/to_change_everything.html.erb` to `app/views/to_change_everything/_share_links.html.erb` either the `yaml` file needs to be updated, or we can just put the full path:

```suggestion
  <li><a href="<%= t('espanol.layouts.to_change_everything.bulk_get.url').html_safe %>"><span class="icon"></span> <%= t('.bulk_get.text').html_safe %></a></li>
```

| path | screenshot |
|------|------|
| https://crimethinc-staging-pr-1426.herokuapp.com/tce/espanol | ![tce1](https://user-images.githubusercontent.com/13190980/71539063-6e3ae300-28eb-11ea-8a26-9cc2b6745e43.png) |